jobs/service-fabrik-backup-manager/templates/config/settings.yml.erb

```diff
@@ -0,0 +1,200 @@
+<%
+  def ostruct_to_hash(os)
+    os.marshal_dump.map do |key, value|
+      { key => value.is_a?(OpenStruct) ? ostruct_to_hash(value) : value }
+    end.reduce(&:merge)
+  end
+
+  broker_name = link("broker").p('name')
+  log_path = "/var/vcap/sys/log/service-fabrik-backup-manager/service-fabrik-backup-manager.log"
+  external = link("broker").p('external')
+  internal = link("broker").p('internal')
+  broker_ip = spec.ip
+%>---
+production:
+  ####################
+  # GENERAL SETTINGS #
+  ####################
+  username: '<%= link("broker").p('username') %>'
+  password: '<%= link("broker").p('password') %>'
+  broker_ip: <%= broker_ip %>
+  enable_service_fabrik_v2: '<%= link("broker").p('enable_service_fabrik_v2') %>'
+  skip_ssl_validation: <%= link("broker").p('skip_ssl_validation') %>
+  session_store:
+    path: <%= "/var/vcap/store/#{broker_name}/session" %>
+  log_path: <%= log_path %>
+  log_level: <%= link("broker").p('log_level') %>
+  enable_circuit_breaker: <%= link("broker").p('enable_circuit_breaker') %>
+  http_timeout: <%= link("broker").p('http_timeout') %>
+  broker_drain_message: <%= link("broker").p('broker_drain_message') %>
+  ##############################
+  # EXTERNAL ENDPOINT SETTINGS #
+  ##############################
+  external:
+    protocol: 'https'
+    host: <%= external['host'] %>
+    cookie_secure: true
+    session_expiry: <%= external['session_expiry'] %>
+
+  ##############################
+  # INTERNAL ENDPOINT SETTINGS #
+  ##############################
+  internal:
+    ip: <%= "#{internal['ip']}" %>
+    host: <%= "#{internal['ip']}:#{internal['port']}" %>
+    <% link("broker").if_p('internal.ssl') do |ssl| %>
+    protocol: 'https'
+    ssl: <%= JSON.dump(ssl) %>
+    <% end.else do %>
+    protocol: 'http'
+    <% end %>
+  ###################
+  # QUOTA MANAGEMENT SETTINGS #
+  ###################
+  quota: <%= JSON.dump(link("broker").p('quota', nil)) %>
+
+  #######################
+  # MONITORING SETTINGS #
+  #######################
+  monitoring:
+    success_state: 'ok'
+    success_metric: 0
+    failure_state: 'critical'
+    failure_metric: 1
+    warning_state: 'warning'
+    inprogress_state: 'in progress'
+    inprogress_metric: 2
+    event_name_prefix: <%= link("broker").p('riemann.prefix') %>.<%= name %>.<%= index %>.<%= spec.deployment %>
```

Comment:
We have created too many links. This way bosh jobs become too much tightly coupled. If in future backup-manager goes out of broker instance groups, then linking would be nightmare. So,  better to have individual spec for them and share feed same value in manifest property. But for this PR we can go with this. Change would huge otherwise.